frontend/note.js

```diff
@@ -67,6 +73,17 @@ class Note {
         this.#editor.remove();
         this.#notelist.remove(this);
     }
+
+    applyFilter(filter) {
+        const name = this.#name.toLowerCase();
+        const content = this.#name.toLowerCase();
```

Comment:
Was this ment to get name? I think content would be what you wanted to get?
```suggestion
        const content = this.#content.toLowerCase();
```